api/organization/update.js

```diff
@@ -6,9 +6,20 @@ const { MSGS: { INTERNAL_SERVER_ERROR, INSUFFICIENT_PERMISSIONS } } = require('.
  * be isolated.
  */
 module.exports = async (req, res, ctx) => {
+  const createStripeCustomerAbstracted = async (email, org) => {
```

Comment:
can we move this func decl to outside the route handler ? so that it doesn't get redeclared each time the route is hit. and pass `ctx` in as a param (or pass in just `stripe` and `db`, along with email and org).

Reply:
just removed it, wasn't saving that many lines